### PR TITLE
Properly uninstall versions.

### PR DESCRIPTION
**Pull request by MisteFr**

### CODE REVIEW

File: MCLauncher/MainWindow.xaml.cs [hunk 1]
```diff
@@ -230,25 +230,25 @@ public partial class MainWindow : Window, ICommonVersionCommands {
...
-                        Proc.Kill();
-                    }
-                }
-                Thread.Sleep(1000);
-
                 v.UpdateBeingRemoveStatus(true);
                 foreach (var pkg in new PackageManager().FindPackages(MINECRAFT_PACKAGE_FAMILY)) {
                     if (pkg.InstalledLocation.Path.Contains(v.GameDirectory)) {
+                        foreach (Process Proc in Process.GetProcesses()) {
+                            if (Proc.ProcessName.Contains("Minecraft")) {
```
Comment (dktapps) [inline MCLauncher/MainWindow.xaml.cs]: what happens if MCJE is running?

Comment (AtomicBlom) [reply MCLauncher/MainWindow.xaml.cs]: Java edition's process name is generally either "java" or "javaw". Not sure about the launcher though.

Comment (AtomicBlom) [reply MCLauncher/MainWindow.xaml.cs]: Java edition's process name is generally either "java" or "javaw". Not sure about the launcher though.

Comment (MCMrARM) [reply MCLauncher/MainWindow.xaml.cs]: killing random processes that happen to include "Minecraft" in the name by pure chance is probably a bad idea

Comment (GrelTheL) [reply MCLauncher/MainWindow.xaml.cs]: I know this is a necropost but why not just kill Minecraft.Windows.exe?

**Comment by dktapps** [comment]
This has already been fixed since the PR was made.

**Pull request closed without merging**